Update README.md

sahaja-mahanty opened the pull request:
Project name: pyOWM
Project website/repository: https://github.com/csparpa/pyowm
License: MIT license

Verified everything in the checklist.
Included pyOWM library in the weather category. It enables fetching various climate related metrics effortlessly.

jhamman commented:
Thank you for the contribution. The checklist is for others to see. Would you mind adding it back here?

sahaja-mahanty commented:
Sure @jhamman 

License: MIT license

Checklist for new projects:

 The project is open-source and accessible. - **yes**
 Searched the existing entries to make sure this is not a duplicate. - **yes**
 Contains only a single addition (make separate PRs if adding more than one). - **yes**
 Added the link to the bottom of the relevant category. - **yes**
 Add icon from the media/icon/ folder if applicable (like ![Python](media/icon/python.png) ). - **yes**
 Created a new category only if necessary. - **yes** [Didn't have to]
 Checked spelling and grammar. - **yes**
 Removed trailing whitespace and periods. - **yes**
 Confirm the dash – is not a minus -. - **yes**
 Used title-casing (AP style) for project name. - **yes**
